public/editor/scripts/project/project.js

```diff
@@ -259,7 +259,6 @@ define(function(require) {
               });
             }, function(err) {
               if (err) {
-                console.error("Failed to download " + DEFAULT_INDEX_HTML_URL, err);
```

Comment:
No, this still isn't right.  What @gideonthomas asked you to do above is to change this:

```js
}, function(err) {
  if(err) {
    return callback(err);
  }
  callback();
});
```

Into this:

```js
}, callback);
```

The only reason to "wrap" the callback would be if you wanted to do something special (like log or handle the error somehow).  In this case, you just want to pass it on unchanged.  This is a common anti-pattern in node.js callback code.